website/docs/language/expressions/references.mdx

```diff
@@ -121,10 +121,11 @@ to mark the reference as for a data resource.
 * `path.module` is the filesystem path of the module where the expression
   is placed.
 * `path.root` is the filesystem path of the root module of the configuration.
-* `path.cwd` is the filesystem path of the current working directory. In
-  normal use of Terraform this is the same as `path.root`, but some advanced
-  uses of Terraform run it from a directory other than the root module
-  directory, causing these paths to be different.
+* `path.cwd` is the filesystem path of the current working directory. This is
+  an absolute path which includes details about the filesystem structure,
+  and is useful in some advanced cases where Terraform is run from a directory
+  other than the root module directory. We recommend using `path.root` or
```

Comment:
```suggestion
* `path.cwd` is the filesystem path of the original working directory from where you ran Terraform before applying any `-chdir` argument. This path is
  an absolute path that includes details about the filesystem structure.
  It is also useful in some advanced cases where Terraform is run from a directory
  other than the root module directory. We recommend using `path.root` or
```

I incorporated the suggestion from @alisdair (much appreciated) with a slight change to the language to avoid a passive voice style issue. Also suggest saying "This path is an absolute path that includes" to avoid "This" with no subject. Also suggest breaking the sentence starting with "This" into 2 sentences for clarity and to avoid having 1 really long run-on sentence.

Reply:
Thanks! I wasn't able to accept this suggestion via the UI so I pushed a commit adding all of this.

Reply:
Huh. I don't think that commit made it to GitHub, or it was dropped when I just resolved the merge conflict on the branch. I'll apply the changes again.